app/pages/project/finished-banner.cjsx

```diff
@@ -33,7 +34,7 @@ module.exports = React.createClass
     if project.redirect
       Promise.resolve false
     else
-      project.get('workflows').then (allWorkflows) ->
+      getWorkflowsInOrder(project).then (allWorkflows) ->
```

Comment:
20 workflows here will be a really slow call....are they really all active and unfinished?